### ISSUE TITLE
need equivalent of OMPI_Snapshot module for python client

### ISSUE THREAD

**Issue by hppritcha** [open-mpi/mtt#577]
As best I can tell, the only available  MiddlewareGet  plugin is Git.  This isn't sufficient for Open MPI testing.  We need a plugin that can download tarballs from a URL, untar, then build/install.

**Comment by noahv** [comment]
I agree a plugin for this would be useful.  

If you need to run this now we have been using the Shell plugin (its a build plugin but works well):

```
command = sh -e -c "wget -e use_proxy=yes -e 'https_proxy=<your_proxy> ' 'https://asc.llnl.gov/CORAL-benchmarks/Science/qball_r140b.tgz'; tar xvfz qball_r140b.tgz"
```

Note that the build plugings expect the target Makefile to be in TestGet directory; if the tarball included a base dir you have to move the files up one level in the command above.

**Comment by hppritcha** [comment]
closed via #605
